utils/logger.js

```diff
@@ -0,0 +1,27 @@
+const pino = require('pino');
+
+require('../src/config');
+
+let logger;
+
+/* If in Development mode:
+ * Output logs to console (which is by default)
+ * Enable prettyPrint option and translate time from epoch time to local time.
+ * Set log level to LOG_LEVEL environment variable with 'debug' as default level.
+ */
+if (process.env.NODE_ENV === 'development') {
+  logger = pino({
+    prettyPrint: { translateTime: 'SYS: yyyy-mm-dd HH:MM:ss.l ' },
+    level: process.env.LOG_LEVEL || 'debug',
+  });
+} else if (process.env.LOG_FILE !== undefined) {
```

Comment:
Just wondering should we specifically check for `undefined` or following code snippet would be better:

`else if (process.env.LOG_FILE){`

Reply:
I guess the comparison to undefined is not necessary. I'll make it just process.env.LOG_FILE

Reply:
The changes have been made. Thanks for your feedback.